Library/Homebrew/global.rb

```diff
@@ -5,6 +5,7 @@
 require "extend/ARGV"
 require "PATH"
 require "extend/string"
+require "vendor/backports/string"
```

Comment:
Maybe worth requiring this inside `require "extend/string"` and adding a comment in there noting we're extending string in another place?